packages/neutrino/src/webpack/utils.js

```diff
@@ -0,0 +1,24 @@
+module.exports = {
```

Comment:
Seeing this makes me wish we had ES modules here. _sigh_

Maybe we can save some nesting by just naming them:

```js
module.exports.handle = /* ... */

module.exports.logStats = /* ... */

module.exports.logErrors = /* ... */
```

We could also use object shorthand to reduce nesting:

```js
function handle() {}

function logStats() {}

function logErrors() {}

module.exports = {
  handle,
  logStats,
  logErrors
};
```